t/rose-task-run/31-app-bunch.t

```diff
@@ -163,5 +163,59 @@ for KEY in $(seq 0 3); do
         "ROSE_BUNCH_LOG_PREFIX: $KEY" $FILE_DIR/bunch.$KEY.out
 done
 #-------------------------------------------------------------------------------
+#-------------------------------------------------------------------------------
+# Testing ROSE_BUNCH_ARGUMENT_MODE_IZIP shortens all bunch-args
+#-------------------------------------------------------------------------------
+APP=bunch_argument_mode_izip
+#-------------------------------------------------------------------------------
+TEST_KEY_PREFIX=argument_mode_izip
+FILE=$LOG_DIR/$APP/01/job.out
+ARG1=1
+ARG2=a
+ARG3=9
+file_grep "$TEST_KEY_PREFIX-RUN-$ARG1-$ARG2-$ARG3" \
+    "\[INFO\] [0-9]*-[0-9]*-[0-9]*T[0-9]*:[0-9]*:[0-9]*+[0:9]* echo arg1: $ARG1 - arg2: $ARG2 - arg3: $ARG3"\
+    "$FILE"
+file_grep "$TEST_KEY_PREFIX-TOTAL-RAN" \
+    "\[INFO\] [0-9]*-[0-9]*-[0-9]*T[0-9]*:[0-9]*:[0-9]*+[0:9]* TOTAL: 1" \
+    "$FILE"
+#-------------------------------------------------------------------------------
+#-------------------------------------------------------------------------------
+# Testing ROSE_BUNCH_ARGUMENT_MODE_IZIP_LONGEST lengthens all bunch-args
+#-------------------------------------------------------------------------------
+APP=bunch_argument_mode_izip_longest
+#-------------------------------------------------------------------------------
+TEST_KEY_PREFIX=argument_mode_izip_longest
+FILE=$LOG_DIR/$APP/01/job.out
+for VALUES in 1,a,9 2,,23 3,, 4,,; do
+    IFS=, read -r -a ARGS <<< "$VALUES"
+    file_grep "$TEST_KEY_PREFIX-RUN-$ARG1-$ARG2-$ARG3" \
+        "\[INFO\] [0-9]*-[0-9]*-[0-9]*T[0-9]*:[0-9]*:[0-9]*+[0:9]* echo arg1: ${ARGS[0]:-} - arg2: ${ARGS[1]:-} - arg3: ${ARGS[2]:-}"\
```

Comment:
Ideally we can also split this long line, & the same for line 211 (& shorten the first line of 177).

Reply:
Would it be appropriate for me to shift the `\[INFO\] [0-9]*-[0-9]*-[0-9]*T[0-9]*:[0-9]*:[0-9]*+[0:9]*` into a different include file somewhere? It is a standard pattern for looking in log files at the info lines. It appears at least 16 times in the test suite (14 in this file, 2 in `37-app-bunch-rm-old.t`).

I am tempted to move it into the `test_header`, but, I don't think it tends to have known constants in there that could be commonly used across tests. For now I'll just put it as a constant at the top of this file itself, unless advised otherwise.

Reply:
Oh, the datetime component of that strings appears more, and in some other test files, it is `\\[INFO\\]....`

Reply:
> It is a standard pattern for looking in log files at the info lines. It appears at least 16 times in the test suite

I agree it would be a good idea to move the pattern somewhere where it can be referenced to prevent duplication. It may be best to, instead of a solitary constant in the ``test_header`` as you are reluctant to add, creating an option to the ``file_grep()`` function that will adjust the behaviour of that test method accordingly in the common case of testing for log line macthes?

I'll get this PR reviewed as-is, without the above change. However, given it is distinct to the core of this PR, I suggest we get this merged as-is & then advise you to put in a separate PR for cleaning up the test suite as above, should you still wish to (it would be great if you did!).